src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java

```diff
@@ -54,31 +63,56 @@ public void entryOption(final String option) {
         }
     }
 
+    /**
+     * Method for remove or load the pot
+     */
+    public void movingPot(final boolean wantPutPot) {
+        if (coffeeMaker.isPotInPlace() && !wantPutPot) {
+            actionPot("Removing the pot ...", "The pot has been removed.");
```

Comment:
It is not necessary to create another class just for the message, you could directly use the output class in this place

Reply:
ok